packages/components/package.json

```diff
@@ -40,7 +40,6 @@
         "find-types": "tsx scripts/findImportedTypes.ts"
     },
     "peerDependencies": {
```

Comment:
in the src/index.ts file, there is a line "export * from "@hopper-ui/styled-system". Now that this package is not a peerDependency, nor a dependency, how will the client know that this package needs to be installed? 

If it works at the moment, i would assume it's because you're most likely bundling both components and styled-system as part of the same package, which isn't the intended approach. 

I would explore more potential solution, as this doesn't look like an acceptable one

Reply:
We're installing "@hopper-ui/icons" as dependency which has "@hopper-ui/styled-system" as a peerDependency, not sure what created the issue.